src/zoid/venmo/component.jsx

```diff
@@ -161,6 +166,19 @@ export function getVenmoCheckoutComponent(): VenmoCheckoutComponent {
           value: getLocale,
         },
 
+        channel: {
+          type: "string",
+          queryParam: true,
+          required: false,
+          value: () => (isDevice ? CHANNEL.MOBILE : CHANNEL.DESKTOP),
```

Comment:
Is `isDevice` a function? If so, we need to execute it.
```suggestion
          value: () => (isDevice() ? CHANNEL.MOBILE : CHANNEL.DESKTOP),
```